app/src/main/java/org/mozilla/fenix/components/metrics/MetricsStorage.kt

```diff
@@ -33,13 +36,20 @@ internal class DefaultMetricsStorage(
     private val settings: Settings,
     private val checkDefaultBrowser: () -> Boolean,
     private val shouldSendGenerally: () -> Boolean = { shouldSendGenerally(context) },
+    private val getInstalledTime: () -> Long = { getInstalledTime(context) },
```

Comment:
these are neat lambdas. Are the default lambdas overridden anywhere to necessitate their parameterization?

Reply:
They are overridden when under test

Reply:
That's my bad. I misinterpreted the Android Studio tooling when trying to cross-check its usages. Carry on